### ISSUE TITLE
Uncaught TypeError: this.escape is not a function

### ISSUE THREAD

**Issue by himanshugpt** [nikhilkalige/docblockr#156]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.3.2
**System**: Mac OS X 10.11.2
**Thrown From**: [docblockr](https://github.com/nikhilkalige/docblockr) package, v0.7.3
### Stack Trace

Uncaught TypeError: this.escape is not a function

```
At /Users/hgupta/.atom/packages/docblockr/lib/languages/java.js:87

TypeError: this.escape is not a function
    at DocsParser.JavaParser.format_function (/Users/hgupta/.atom/packages/docblockr/lib/languages/java.js:87:84)
    at DocsParser.parse (/Users/hgupta/.atom/packages/docblockr/lib/docsparser.js:44:37)
    at DocBlockrAtom.module.exports.DocBlockrAtom.parse_command (/Users/hgupta/.atom/packages/docblockr/lib/docblockr-worker.js:279:29)
    at atom-workspace.<anonymous> (/Users/hgupta/.atom/packages/docblockr/lib/docblockr-worker.js:75:16)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Users/hgupta/Desktop/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Users/hgupta/Desktop/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Users/hgupta/Desktop/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Users/hgupta/Desktop/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Users/hgupta/Desktop/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Users/hgupta/Desktop/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```
### Commands

```
     -1:38.3.0 core:paste (atom-text-editor.editor.is-focused)
     -1:35.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:33.3.0 core:move-down (atom-text-editor.editor.is-focused.autocomplete-active)
     -1:33.1.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -1:31.4.0 core:move-right (atom-text-editor.editor.is-focused)
     -1:30.4.0 core:save (atom-text-editor.editor.is-focused)
  2x -0:34.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:33.2.0 core:save (atom-text-editor.editor.is-focused)
     -0:27.9.0 core:copy (atom-text-editor.editor.is-focused)
     -0:26.4.0 core:paste (atom-text-editor.editor.is-focused)
     -0:23.2.0 core:move-right (atom-text-editor.editor.is-focused)
  3x -0:20.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:17.2.0 core:save (atom-text-editor.editor.is-focused)
     -0:15.3.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:15.3.0 editor:newline (atom-text-editor.editor.is-focused)
  3x -0:14.3.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
autocomplete-go, v0.1.1
autocomplete-python, v1.2.4
build, v0.52.0
build-gradle, v0.6.0
docblockr, v0.7.3
environment, v1.1.1
go-config, v0.3.1

# Dev
No dev packages
```

**Comment by MoritzKn** [comment]
Is this still an Issue? If yes, please explain how to reproduce this.

**Comment by himanshugpt** [comment]
Just close it. I moved to sublime and no idea if its still happening or not.

**Comment by maxwey** [comment]
I got this issue on two separate machines when attempting to add documentation for a Java method that throws >1  exception.

Such as attempting to add documentation to the following `foo()` method: 
```Java
public void foo() throws FileNotFoundException, IOException {
   ...
}
```

_Also this seems to happen only when using `/**` and not `/*`_

Please let me know if you need any more information about my machine/stacktrace/other.

**Comment by MoritzKn** [comment]
@maxwey thanks for reporting this and the accurate description. The fix for this was just released with the version v0.8.8.
